src/aura/CMP_RecTypes_Dropdown/CMP_RecTypes_Dropdown.cmp

```diff
@@ -1,22 +1,23 @@
 <aura:component >
     <aura:handler event="c:STG_EVT_Save" action="{!c.saveSetting}"/>
-    
+
     <aura:attribute name="isView" type="Boolean" default="true"/>
     <aura:attribute name="recTypeName" type="String" />
     <aura:attribute name="recTypeId" type="String" />
     <aura:attribute name="recTypes" type="Object[]" />
+    <aura:attribute name="dropdownDisabled" type="boolean" default="false" />
 
     <aura:attribute name="setting" type="String" />
-    
+
     <aura:if isTrue="{!v.isView}">
         <ui:outputText aura:id="recTypeName" value="{!v.recTypeName}" class="rec-type-output"/>
     <aura:set attribute="else">
-        <ui:inputSelect aura:id="recTypesDropDown" change="{!c.onSelectChange}">
+        <ui:inputSelect aura:id="recTypesDropDown" change="{!c.onSelectChange}" disabled="{!v.dropdownDisabled}">
```

Comment:
Now that I think about it, it's completely possible the framework has been improved since then, and rendering in the dom is done correctly when using the disabled attribute. It should be easy to test in the browser.